persistence/subsystem/offsetstore/offset.go

```diff
@@ -0,0 +1,4 @@
+package offsetstore
+
+// Offset is the position of an event within an application's event stream.
+type Offset uint64
```

Comment:
```suggestion
```

As per #172 this type should be removed and you should instead reference `eventstream.Offset` directly. I haven't actually *done* anything with #172 yet it's just highlighting a package structure that seems odd.

Reply:
No worries, I can do that in my next commit.